### ISSUE TITLE
Special characters and brackets enclosing

### ISSUE THREAD

**Issue by benoitseron** [FlamingTempura/bibtex-tidy#407]
When using the option "Enclose values in double braces" I found that special characters induce bugs. For instance,

```
      author={Garc{\'\i}a-Patr{\'o}n, Ra{\'u}l and Renema, Jelmer J and Shchesnovich, Valery},
```

becomes 

```
	author       = {{Garc\'\i}a-Patr{\'o}n, Ra{\'u}l and Renema, Jelmer J and Shchesnovich, Valery}}}},
```

I am not sure if this is normal behaviour but it leads to problems at compilation (next lines will usually not show).

**Comment by FlamingTempura** [comment]
Thanks for the report - appears to be related to #410.

This is top of my list to look into when I find some time to work on bibtex-tidy.

**Comment by FlamingTempura** [comment]
Fixed in v1.14.0
